cluster/charts/crossplane/README.md

```diff
@@ -63,13 +63,15 @@ and their default values.
 
 | Parameter | Description | Default |
 | --- | --- | --- |
+| `affinity` | Enable affinity for Crossplane Managers pod | `{}` |
```

Comment:
nitpick: could you strip off "Managers" from the Crossplane-specific fields

Reply:
it's not nitpick - it's not the Crossplane Managers pod.  I just removed it.  I had copied/pasted and that existed in other crossplane pod descriptions (ex. leaderElection.)  I can remove it from those as well if that would make sense, as part of this PR